routes/jobs.js

```diff
@@ -1,11 +1,17 @@
+const dbCommands = require('../model/dbCommands');
 var express = require('express')
 var router = express.Router()
 
-
 router.get('/hello', function (req, res) {
     res.status(200);
     res.send('Hello');
 })
+
+router.get('/job-roles-spec/:Role_ID', async(req, res) =>
```

Comment:
Might be good to add some error handling here https://github.com/Matthew-Kainos/KainosAcademy/pull/2/files in  the routes/capabilities file

Reply:
Added error handling.